cmd/tracetest/main.go

```diff
@@ -42,21 +67,73 @@ func main() {
 		logger.Error("start trace", zap.Error(err))
 	}
 
-	time.Sleep(5 * time.Second)
+	// First Test
+	println("Start First Test")
```

Comment:
use logger

Reply:
the println was so you can see whats going on as a user rather than wait on a black screen